packages/xo-server-sdn-controller/src/index.js

```diff
@@ -231,7 +245,11 @@ class SDNController extends EventEmitter {
 
     // Expose method to create cross-pool private network
     const createCrossPoolPrivateNetwork = params =>
-      this._createCrossPoolPrivateNetwork({ encrypted: false, mtu: 0, ...params })
```

Comment:
Not my modification, I think there's been a prettier update somewhere.

Reply:
I have pushed this change on `master` directly so your PR can stay clean :slightly_smiling_face: